manimlib/camera/camera.py

```diff
@@ -66,34 +88,106 @@ def __deepcopy__(self, memo):
         return copy.copy(self)
 
     def reset_pixel_shape(self, new_height, new_width):
+        """This method resets the height and width
+        of a single pixel to the passed new_heigh and new_width.
+
+        Parameters
+        ----------
+        new_height : int, float
```

Comment:
1. Use Union[a, b] for multiple type possibilities.
2. Use ``:class:`int`, :class:`float` ``